index.html

```diff
@@ -123,7 +123,7 @@
         <p>A deck which contains only Standard cards is called a <i>Standard deck</i>.</p>
         <h3>
           Banlist
-          <a class="tip" title="Despite being printed into Standard sets, these cards are not allowed in Standard decks as of January 2017."></a>
+          <a class="tip" title="Despite being printed into Standard sets, these cards are not allowed in Standard decks as of April 2017."></a>
```

Comment:
I'm not attached to showing when each card was banned, but if you can make it clean and low-distraction I'm also not against it. If you use @zulaica's method make sure people won't get confused and think the months are "ban is active during" time periods.

Linking to sources is always good 👍